packages/hash/api/src/graphql/resolvers/knowledge/user/createUser.ts

```diff
@@ -0,0 +1,64 @@
+import { createKratosIdentity } from "../../../../auth/ory-kratos";
+import { UserModel } from "../../../../model";
+import {
+  MutationCreateUserArgs,
+  ResolverFn,
+  Subgraph,
+} from "../../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+import { mapSubgraphToGql } from "../../ontology/model-mapping";
+
+export const createUser: ResolverFn<
+  Promise<Subgraph>,
+  {},
+  LoggedInGraphQLContext,
+  MutationCreateUserArgs
+> = async (
+  _,
+  {
+    email,
+    password,
+    isInstanceAdmin,
+    shortname,
+    preferredName,
+    linkResolveDepth,
+    linkTargetEntityResolveDepth,
+  },
+  { dataSources: { graphApi }, userModel: actorUserModel },
+) => {
+  const { address: emailAddress, verified: emailAddressVerified } = email;
+
+  const kratosIdentity = await createKratosIdentity({
+    traits: {
+      shortname: shortname ?? undefined,
+      emails: [emailAddress],
+    },
+    credentials: { password: { config: { password } } },
+  });
+
+  if (emailAddressVerified) {
+    /**
+     * @todo: use kratos admin API to programmatically verify the email of the user
+     *
+     * @see https://github.com/ory/kratos/issues/2473
+     */
+  }
```

Comment:
@Alfred-Mountfield @thehabbos007 do you think I should remove the `verified` GQL param for now as we are currently unable to implement it?

Reply:
I'd remove the param.

(Just to make sure we're on the same page though, I'd definitely leave the TODO, just maybe remove the block wrapping it)

Reply:
Yeah, that was my instinct as well. Have done so in https://github.com/hashintel/hash/pull/1360/commits/4abfbad191f55f4ad28b3a84eb61d7a77028c70b